### ISSUE TITLE
Plugin doesn't work, throws "command sftp.config is not found" error

### ISSUE THREAD

**Issue by kolnogorov** [liximomo/vscode-sftp#548]
**Do you read the FAQ?**
- [x] Yes.
- [ ] [I am going to read now.](https://github.com/liximomo/vscode-sftp/blob/master/FAQ.md)

**Describe the bug**
Plugin version 1.12.3, VSCode 1.33.1, macOS Mojave 10.14.4
SFTP doesn't start, can't see config file. When i try to SFTP: Config, it throws error: "command sftp.config is not found".

Reinstalling the plugin and VSCode didn't help.

**To Reproduce**
1. Run VSCode with any (or without) project.
2. SFTP doesn't start and does not have any SFTP-commands in Command palette.
3. The only available command there is SFTP is SFTP: Config, but when chosen, it throws an error "command sftp.config is not found".

**Expected behavior**
SFTP should work.

**Desktop (please complete the following information):**
 - OS: macOS Mojave 10.14.4
 - VSCode Version 1.33.1
 - Extension Version 1.12.3

**Extension Logs from Startup** - *required*
  1.  Open User Settings.
      * On Windows/Linux - File > Preferences > Settings
      * On macOS - Code > Preferences > Settings
  2. Set `sftp.debug` to `true` and reload vscode.
  3. Reproduce the problem, get the logs from View > Output > sftp.

There is no any logs there, just empty.

**Comment by kolnogorov** [comment]
I have installed previous version of plugin (1.12.2) and everything works correct with it.
After updating to 1.12.3 it stops working.

**Comment by liximomo** [comment]
Try a [completely uninstall](https://stackoverflow.com/questions/36746857/completely-uninstall-vs-code-extensions).

**Comment by kolnogorov** [comment]
> Try a [completely uninstall](https://stackoverflow.com/questions/36746857/completely-uninstall-vs-code-extensions).

So, turns out, i should completely uninstall all extensions? I have a plenty of them, that will be tough to restore.

**Comment by kolnogorov** [comment]
I uninstalled just a plugin folder, and the new error appeared.
When i do Sync Local > Remote, vscode is showing a "Failure" popup error.

```
[05-06 12:24:19] [error] Error: Failure
	at SFTPStream._transform (/Users/lovebeam/.vscode/extensions/liximomo.sftp-1.12.4/node_modules/ssh2-streams/lib/sftp.js:412:27)
	at SFTPStream.Transform._read (_stream_transform.js:190:10)
	at SFTPStream._read (/Users/lovebeam/.vscode/extensions/liximomo.sftp-1.12.4/node_modules/ssh2-streams/lib/sftp.js:183:15)
	at SFTPStream.Transform._write (_stream_transform.js:178:12)
	at doWrite (_stream_writable.js:410:12)
	at writeOrBuffer (_stream_writable.js:394:5)
	at SFTPStream.Writable.write (_stream_writable.js:294:11)
	at Channel.ondata (_stream_readable.js:672:20)
	at Channel.emit (events.js:182:13)
	at Channel.EventEmitter.emit (domain.js:442:20)
	at addChunk (_stream_readable.js:279:12)
	at readableAddChunk (_stream_readable.js:264:11)
	at Channel.Readable.push (_stream_readable.js:219:10)
	at SSH2Stream.<anonymous> (/Users/lovebeam/.vscode/extensions/liximomo.sftp-1.12.4/node_modules/ssh2/lib/Channel.js:167:15)
	at SSH2Stream.emit (events.js:182:13)
	at SSH2Stream.EventEmitter.emit (domain.js:442:20)
	at parsePacket (/Users/lovebeam/.vscode/extensions/liximomo.sftp-1.12.4/node_modules/ssh2-streams/lib/ssh.js:3295:10)
	at SSH2Stream._transform (/Users/lovebeam/.vscode/extensions/liximomo.sftp-1.12.4/node_modules/ssh2-streams/lib/ssh.js:693:13)
	at SSH2Stream.Transform._read (_stream_transform.js:190:10)
	at SSH2Stream._read (/Users/lovebeam/.vscode/extensions/liximomo.sftp-1.12.4/node_modules/ssh2-streams/lib/ssh.js:252:15)
	at SSH2Stream.Transform._write (_stream_transform.js:178:12)
	at doWrite (_stream_writable.js:410:12)
	at writeOrBuffer (_stream_writable.js:394:5)
	at SSH2Stream.Writable.write (_stream_writable.js:294:11)
	at Socket.ondata (_stream_readable.js:672:20)
	at Socket.emit (events.js:182:13)
	at Socket.EventEmitter.emit (domain.js:442:20)
	at addChunk (_stream_readable.js:279:12)
	at readableAddChunk (_stream_readable.js:264:11)
	at Socket.Readable.push (_stream_readable.js:219:10)
	at TCP.onread (net.js:636:20) 
[05-06 12:24:19] [debug] Outgoing: Writing OPENDIR
[05-06 12:24:19] [debug] Outgoing: Writing CHANNEL_DATA (0)
[05-06 12:24:19] [debug] Outgoing: Writing OPENDIR
[05-06 12:24:19] [debug] Outgoing: Writing CHANNEL_DATA (0)
[05-06 12:24:19] [debug] Outgoing: Writing OPENDIR
```

And then the latest strings:
```
[05-06 12:24:19] [debug] Outgoing: Writing CHANNEL_DATA (0)
[05-06 12:24:19] [debug] Outgoing: Writing OPENDIR
```
are repeating endlessly.

I tried to rollback plugin to 1.12.3 version, and got the same error.

**Comment by avtarnanrey** [comment]
I installed it on the same configuration as @kolnogorov has. For me "No Commands Matching".

**Comment by AlecKC** [comment]
This issue was resolved for me when I updated VS Code to 1.35.1.
Sadly I do not know which version I had before the update, but it was MANY months old.  Here is the about info of my VS Code version and system

Version: 1.35.1 (system setup)
Commit: c7d83e57cd18f18026a8162d042843bda1bcf21f
Date: 2019-06-12T14:30:02.622Z
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763

**Comment by TrueMeltix** [comment]
I updated VS Code to 1.40.0, and even after this update, SFTP doesn't work. If I select the option SFTP: Config, the sftp.config appears again.
